### ISSUE TITLE
Python Bindings: -lboost_python-mt not found (already checked troubleshooting)

### ISSUE THREAD

**Issue by martha-thomae** [DDMAL/libmei#104]
This issue happened using either python2 or python3. Using the libmei code from the **develop branch** (**and** from the **previous release, the 2.0.0 version**) when I do `python setup.py build` I got the message:
> Library not found for -lboost_python-mt

Even though I did `export CFLAGS="-F /Library/Frameworks -I /usr/local/include -L /usr/local/lib"` to tell the system where lboost_python-mt is, as indicated in the **Troubleshooting** section of the **python bindings** in the wiki.

**Comment by martha-thomae** [comment]
In `setup.py` where you have:
```
if sys.platform == "darwin":
    link_args = ["-F", "/Library/Frameworks","-framework", "mei"]
    libraries = ["boost_python-mt"]
    library_dirs = []
    runtime_library_dirs = []
    include_dirs = []
```
Try using `libraries = ["boost_python27-mt"]` instead. Since I changed computers, I needed to install boost-python again, and I think that the new version of boost-python is a little different. You don't have something called `boost_python-mt` anymore, but `boost_python27-mt` and `boost_python36-mt`.
Let me know how it turns out. I used `libraries = ["boost_python27-mt"]` and it works. I wasn't able to make it work for python3 though (using `libraries = ["boost_python36-mt"]` didn't work).
